services/QuillLMS/db/migrate/20220505154951_create_response_prompt.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateResponsePrompt < ActiveRecord::Migration[5.1]
+  def change
+    create_table :response_prompts, id: false do |t|
+      t.serial :id, limit: 4, primary_key: true
```

Comment:
What's the reason for using `t.serial` here?

Reply:
Nevermind. I use `create_table :response_prompts, id: :integer` and then `:bigint` for `limit: 8`. I had not seen `t.serial` before.

Reply:
I hadn't either until I was trying to figure out how to specify auto incremented IDs directly.  The internet taught me.